app/api/event_tag_endpoint.py

```diff
@@ -1,26 +1,31 @@
 from tracardi.service.storage.drivers.elastic.tag import add_tags, remove_tags, load_all_tags
 from pydantic import BaseModel
 from typing import List
-from fastapi import APIRouter
+from fastapi import APIRouter, Depends, HTTPException
 from app.config import server
 from tracardi.domain.value_object.bulk_insert_result import BulkInsertResult
+from .auth.authentication import get_current_user
 
 
 class TagForm(BaseModel):
```

Comment:
This is a domain object and should be in Tracardi instead of API. And should have a domain name: `EventTag`

Reply:
Move it to tracardi domain and see Tracardi PR how it is used in driver to simplify validation and adding new tags.